README.md

```diff
@@ -122,6 +122,27 @@ Only the client devices have information about users, their contacts and groups.
 
 See [SimpleX whitepaper](https://github.com/simplex-chat/simplexmq/blob/master/protocol/overview-tjr.md) for more information on platform objectives and technical design.
 
+## Privacy: technical details and limitations
+
+SimpleX Chat is a work in progress, we are releasing improvements as they are ready. You have to decide if the current state is good enough for your scenarios.
+
+What is already implemented:
+
+1. Instead of user profile identifiers used by all other platforms, even the most private, SimpleX uses pairwise per-queue identifiers (2 addresses for each message queue, 2 queues in each connection between the users) . It makes understanding of network graph more difficult on the application level, as for `n` users you have `n * (n-1)` message queues.
```

Comment:
```suggestion
1. Instead of user profile identifiers used by all other platforms, even the most private, SimpleX uses pairwise per-queue identifiers (2 addresses for each message queue, 2 queues in each connection between the users). It makes understanding of network graph more difficult on the application level, as for a complete graph of `n` users you have `n * (n-1)` message queues.
```

Reply:
should be fully connected, but better "you can have up to"... will change